test/e2e/iframe3/index.html

```diff
@@ -0,0 +1,19 @@
+<head>
+  <script src="/dist/index.min.js"></script>
+  <script src="/analytics.js"></script>
+</head>
+
+<body>
+  <h1 id="h1">Hello world!</h1>
+
+  <script>
+    // <iframe src="/stub.html?delay=500"></iframe>
+    const iframe = document.createElement('iframe');
+    // "src" is technically a required field in HTML iframes. Many libraries work around
+    // this by specifying the url "about:blank".  we should handle this scenario.
+    iframe.src = 'about:blank';
```

Comment:
From https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/src
> Note that programmatically removing an <iframe>'s src attribute (e.g. via [Element.removeAttribute()](https://developer.mozilla.org/en-US/docs/Web/API/Element/removeAttribute)) causes about:blank to be loaded in the frame.

Do you think it's worth having a test that simulates this (in other words, programmatically remove the src attribute)?

Reply:
I did actually try this manually.  In all of the browsers I have installed, the contents of the iframe may be `about:blank`, but the src attribute itself remains empty.

So I think this finding was a bit of a red herring on my part, and our existing code already handles that well.  🤷